src/Controller/RegistrationController.php

```diff
@@ -78,7 +78,7 @@ public function __invoke(Request $request): Response
         return $this->render(
             'default\registration.html.twig',
             [
-                'publicKeyOptions' => $publicKeyCredentialCreationOptions
+                'userEntity' => $userEntity
```

Comment:
For future improvement: we can get the required user information from the AuthnRequest that is sent to the registration endpoint. This contains a NameId (username). Using that username also defeats the need for the user input at [templates/default/registration.html.twig](https://github.com/OpenConext/Stepup-Webauthn/pull/113/files#diff-456ae19358573f314fed0f5cfe259a1cec584819e28b9c9a08a5438c0e4685a6)

**Edit 18-3 10:32** I later observed the aforementioned twig template already handles the registration without user interaction.